modules/client-configuration/pages/delete-clients.adoc

```diff
@@ -19,14 +28,179 @@ For more information about the system set manager, see xref:client-configuration
 
 [IMPORTANT]
 ====
-It is not possible to automatically clean up a a traditional client after deleting it. You have to take care of this
-yourself. Furthermore cleaning up a Salt minion does not remove Salt itself.
+It is not possible to automatically clean up a traditional client after deleting it.
+You have to take care of this yourself.
+Furthermore cleaning up a Salt minion does not remove Salt itself.
 ====
 
 [NOTE]
 ====
-Normally you migrate a traditional client to a Salt minion without deleting the client. 
-Salt automatically detects that you have a traditional client and does the necessary changes itself. 
+Normally you migrate a traditional client to a Salt minion without deleting the client.
+Salt automatically detects that you have a traditional client and does the necessary changes itself.
 But if you already deleted the traditional client and want to register it as a Salt minion again, see
 xref:administration:troubleshooting/tshoot-register-trad-as-salt-after-deletion.adoc[Troubleshooting Registering a traditional client as Salt minion after deleting it].
 ====
+
+
+
+== Delete a Salt Client on the Command Line (with API Call)
+
+.Procedure: Deleting a Client from the Server
+
+. Delete the client with the FQDN (Fully Qualified Domain Name):
++
+----
+spacecmd system_delete FQDN
+----
+
+. Remove the Salt key with the FQDN:
++
+----
+salt-key -d FQDN
+----
+
+[NOTE]
+====
+For specific cases, the new registration of a deleted (unregistered) client might not be possible.
+To solve this issue, some Salt files should be deleted on the {productname} Server (Salt master) before trying to re-register again:
+
+----
+rm /var/cache/salt/master/thin/version
+rm /var/cache/salt/master/thin/thin.tgz
+----
+====
+
+
+
+
+[[delete.clients.commandline]]
+== Delete a Client from the Command Line
+
+
+=== Salt Client
+
+// Manual Registration Cleanup
+
+NOTE: This process is only for {productname} clients, do not run it on the {productname} Server itself.
+
+[NOTE]
+====
+You must not execute the following procedure on clients  running {rhel} and clones.
+====
+
+.Procedure: Salt Client
+
+. Stop the salt-minion service:
++
+SLES 12/15:
++
+----
+systemctl stop salt-minion
+----
++
+SLES 11:
```

Comment:
Do we still add content relevant for SLES 11? The references have been or are being removed from other places/books.

Reply:
yes, but I thought it could be useful if we'd mention the most recent unsupported version here.  Maybe, the user just wants to delete it because it is unsupported now.  I'm not sure, maybe it is less confusing if we do not mention the unsupported client even here.

Reply:
I dropped sles 11, you are right.